_ideas/2020-06-26-add-a-root-pythonization.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Make PyROOT more pythonic
+summary: Add pythonizations for ROOT classes, which make it easier to use ROOT from Python.
+layout: single
+author: Enric Tejedor
+state:
+
+sidebar:
+  nav: "contribute"
+---
+
+ROOT's Python bindings, called [PyROOT](https://root.cern/manual/python), allow to access all the ROOT C++ functionality
```

Comment:
Absolute link like that will not work. 
`[PyROOT](https://root.cern/manual/python)` should be something like:
`[PyROOT]({{'manual/python' | relative_url}})`